Woof/Woof/ViewModel/LoginViewModel.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+/// An interface providing for `the LoginView` to interact with the model layer.
+final class LoginViewModel: ObservableObject {
```

Comment:
Why do we need toconform to `ObservableObject`? What property is observed?

Reply:
you right
i'll delete this